manifests/supervisorcluster/1.30/cns-csi.yaml

```diff
@@ -78,6 +78,12 @@ rules:
   - apiGroups: ["cns.vmware.com"]
     resources: ["storagepolicyusages/status"]
     verbs: ["update", "patch"]
+  - apiGroups: ["cns.vmware.com"]
```

Comment:
Do we need to copy it to previous 2 versions' files as well?

Reply:
seems not necessary as New CR is being introduced from 1.30